great_expectations/expectations/metrics/column_aggregate_metrics/column_distinct_values.py

```diff
@@ -30,9 +33,18 @@ def _sqlalchemy(
         metric_value_kwargs: Dict,
         metrics: Dict[str, Any],
         runtime_configuration: Dict,
-    ):
-        observed_value_counts = metrics["column.value_counts"]
-        return set(observed_value_counts.index)
+    ) -> Set[Any]:
+        selectable, _, accessor_domain_kwargs = execution_engine.get_compute_domain(
+            metric_domain_kwargs, MetricDomainTypes.COLUMN
+        )
+        column = accessor_domain_kwargs["column"]
+
+        query = sa.select(sa.column(column)).distinct()
+        results = execution_engine.engine.execute(
```

Comment:
What are some example queries here? Is this returning all the distinct values for a column?

Reply:
Yes that is correct. `SELECT DISTINCT(column) FROM table`